ci-operator/step-registry/workers-rhel/pre-hook/gcp/workers-rhel-pre-hook-gcp-commands.sh

```diff
@@ -20,12 +20,22 @@ cat > scaleup-pre-hook-gcp.yaml <<- 'EOF'
 - name: Configure RHEL machine on GCP
   hosts: new_workers
   any_errors_fatal: true
-  gather_facts: false
+  gather_facts: true
 
   tasks:
+  - name: cache repo metadata in advance
+    shell: |
+      sudo {{ ansible_pkg_mgr }} clean all
```

Comment:
@jianli-wei Curious why we need `sudo` here, I noticed we already have `ansible_become=True` set in the ansible inventory file, 

https://github.com/openshift/release/blob/master/ci-operator/step-registry/workers-rhel/gcp-provision/workers-rhel-gcp-provision-commands.sh#L74-L75